src/rcPresenceFail/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Danger message failure.
+ * @constant
+ * @type {String}
+ * @default
+ */
+const MESSAGE = `<b>Cannot merge release candidate!</b> - <i>
+It seems you forgot to remove "-rc" from your package.json file.
+</i> 🧐`;
+
+/**
+ * Return true if detect "-rc" in package.json.
+ * @param {String} str
+ * @returns {Boolean}
+ */
+const rcDetected = (str) =>
+    str && str.includes('-rc');
```

Comment:
I'm suggesting you check there's also a "version" in the line
```suggestion
    str && /version(.*)-rc/.test(str);
```

Reply:
👍

Reply:
yes, the diff contains the entire line 👍